### DISCUSSION TITLE
for help,to get data for loopdata,but not work

### DISCUSSION THREAD

**Discussion by techflag** [AutomaApp/automa#364] [Q&A]
<img width="581" alt="image" src="https://user-images.githubusercontent.com/939362/154794794-ffc52754-99b2-4fc1-bf4c-9d0847bc675a.png">

**[preblockdata]**NO-1
`
jobs_arr = [ { "index": "1", "Tagline": "Be less busy. Real-time messaging, archiving & search.", "Tagline_link": "https://www.producthunt.com/posts/slack" }, { "index": "2", "Tagline": "Free (do whatever you want) high-resolution photos.", "Tagline_link": "https://www.producthunt.com/posts/unsplash-5-0" }, { "index": "3", "Tagline": "A new way to start an internet business anywhere", "Tagline_link": "https://www.producthunt.com/posts/stripe-atlas-pre-launch" }, { "index": "4", "Tagline": "Build a fully functional web app without any code", "Tagline_link": "https://www.producthunt.com/posts/bubble-3" }, { "index": "5", "Tagline": "The long awaited version 3 of the popular code editor", "Tagline_link": "https://www.producthunt.com/posts/sublime-text-3-0" }, { "index": "6", "Tagline": "An AI assisstant that can talk on the phone 🤖", "Tagline_link": "https://www.producthunt.com/posts/google-duplex" }, { "index": "7", "Tagline": "Discover what runs a website in one click  🖥️ 🔍", "Tagline_link": "https://www.producthunt.com/posts/whatruns" }, { "index": "8", "Tagline": "Illustrations for everyone", "Tagline_link": "https://www.producthunt.com/posts/blush-3" }, { "index": "9", "Tagline": "A simple, meaningful way to fund your creativity.", "Tagline_link": "https://www.producthunt.com/posts/buy-me-a-coffee" } ];

automaNextBlock({jobs_arr:jobs_arr})`

NO-3:
<img width="937" alt="image" src="https://user-images.githubusercontent.com/939362/154794839-11329c57-04d7-4e58-8916-a938da3efdce.png">

can not get "Tagline_link" from loopdata

**Comment by Kholid060** [comment]
Try `{{ loopData@mfnJ0C.Tagline_link }}`
